competiciones/jornada.py

```diff
@@ -0,0 +1,10 @@
+from dataclasses import dataclass
+
+@dataclass
+class Jornada:
+
+    # Debe de ser un pair que incluya la clase enfrentamiento y el resultado (1, X o 2)
+    enfrentamientos : list
+
+    def __init__(self):
+        self.enfrentamiento = []
```

Comment:
Este no es el atributo declarado. Por favor, tened cierto cuidado cuando hacéis las cosas